scripts/rollup/packaging.js

```diff
@@ -85,7 +87,7 @@ async function createFacebookWWWBuild() {
 }
 
 async function copyBundleIntoNodePackage(packageName, filename, bundleType) {
-  const packageDirectory = resolve(`./build/packages/${packageName}`);
+  const packageDirectory = resolve(`./build/.tmp/${packageName}`);
```

Comment:
What do you think about using a folder in `os.tmpdir()` here instead? Not sure if it would be better, but IMO it's easier to understand if there are fewer files being moved around inside the `build` folder.

Reply:
Agree, updated to use `os.tmpdir()`, it makes sense to do **temp** packing and unpacking in the os's temporary folder.
There may be some rare cases, if a use's permission is misconfigured, system temporary folder is not accessible for writing.
Please review.

Reply:
>There may be some rare cases, if a use's permission is misconfigured, system temporary folder is not accessible for writing.

Yeah. I think let's risk it and see if it's an issue in practice or not.